src/providers/toast.service.ts

```diff
@@ -14,4 +19,9 @@ export class ToastService {
     this.faveToast = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
     this.faveToast.present();
   }
+  noOriginOrDestinationToast(): void{
```

Comment:
Question: is this needed here? It looks like you changed the code to use `toastHandler` instead.

Reply:
[(https://github.com/umts/pvta-multiplatform/blob/master/src/pages/plan-trip/plan-trip.component.ts#L296-L299)]

Reply:
this one behaves differently than the other toasts

Reply:
I see. Although that code won't do anything anymore right? Should that be fixed?

Reply:
what do u mean by it will no longer work??

Reply:
Sorry, let me clarify. I agree that it is a different toast because we have code that wants to grab hold of it to dismiss it. But, now it is never `present`ed - that was [removed here](https://github.com/umts/pvta-multiplatform/pull/434/files#diff-8f802a9b80a60c0a01b7ed67374a46b8L315).

Reply:
woops I tried to extract it out realized it wont work and didnt put it back.. I fixed it :)